crates/bvs-vault-router/src/contract.rs

```diff
@@ -325,13 +326,135 @@ mod execute {
                     .add_attribute("reason", data.metadata.reason),
             ))
     }
+
+    pub fn slash_locking(
+        deps: DepsMut,
+        env: Env,
+        info: MessageInfo,
+        id: state::SlashingRequestId,
+    ) -> Result<Response, ContractError> {
+        let registry = state::get_registry(deps.storage)?;
+
+        let slash_req = state::SLASHING_REQUESTS.may_load(deps.storage, id.clone())?;
+
+        if slash_req.is_none() {
+            return Err(ContractError::InvalidSlashingRequest {
+                msg: "Slashing entry with given id does not exist".to_string(),
```

Comment:
I think the error will already have "Invalid slashing request: {msg}" appended. so maybe just a simple
```suggestion
                msg: "Id does not exists".to_string(),
```